frontend/package.json

```
@@ -1,38 +1,42 @@
 {
-	"name": "kakaw",
-	"version": "0.1.0",
-	"private": true,
-	"scripts": {
-		"dev": "next dev",
-		"build": "next build",
-		"start": "next start",
-		"lint": "next lint",
-		"test": "jest --coverage",
-		"format": "prettier -w pages public styles components"
-	},
-	"dependencies": {
-		"@types/node": "18.15.11",
-		"@types/react": "18.0.37",
-		"@types/react-dom": "18.0.11",
-		"autoprefixer": "10.4.14",
-		"eslint": "8.38.0",
-		"eslint-config-next": "13.3.0",
-		"next": "13.3.0",
-		"postcss": "8.4.22",
-		"react": "18.2.0",
-		"react-dom": "18.2.0",
-		"tailwindcss": "3.3.1",
-		"typescript": "5.0.4"
-	},
-	"devDependencies": {
-		"@testing-library/jest-dom": "^5.16.5",
-		"@testing-library/react": "^14.0.0",
-		"@testing-library/user-event": "^14.4.3",
-		"@types/jest": "^29.5.0",
-		"@types/testing-library__jest-dom": "^5.14.5",
-		"identity-obj-proxy": "^3.0.0",
-		"jest": "^29.5.0",
-		"jest-environment-jsdom": "^29.5.0",
-		"prettier": "^2.8.7"
-	}
+
+  "name": "kakaw",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint",
+    "test": "jest --coverage"
```

```suggestion
    "test": "jest --coverage",
```

npm doesn't parse this without the comma